### PR TITLE
Improve unreasonable formatting in documents

### PR DESCRIPTION
**Pull request by acmenlt**
Changes proposed in this pull request:
- Improve unreasonable formatting in documents

### CODE REVIEW

File: docs/community/content/contribute/document-contributor.cn.md [hunk 1]
```diff
@@ -98,20 +98,20 @@ ShardingSphere 文档使用 Markdown 编写，并使用 Hugo 进行处理生成
 
 1. 附加出现问题的文档链接。
 
-1. 详细描述问题。
+2. 详细描述问题。
```
Comment (taojintianxia) [inline docs/community/content/contribute/document-contributor.cn.md:101]: as a matter of fact, this is not "an error", github markdown engine will render the document to what we expected even the order of number are all 1.

anyway, thanks for your contribution to "fix" this.

File: docs/document/content/dev-manual/scaling.cn.md [hunk 2]
```diff
@@ -46,5 +46,5 @@ chapter = true
 
 | *已知实现类*                                  | *详细说明*                                                              |
 | ------------------------------------------- | ---------------------------------------------------------------------- |
-| DataMatchSingleTableDataCalculator          | 给 DATA_MATCH 数据一致性校验算法使用的单表数据计算算法。适用于所有数据库。        |
-| CRC32MatchMySQLSingleTableDataCalculator    | 给 CRC32_MATCH 数据一致性校验算法使用的单表数据计算算法。适用于 MySQL。          |
+| DataMatchSingleTableDataCalculator          | 给 DATA_MATCH 数据一致性校验算法使用的单表数据计算算法。适用于所有数据库        |
+| CRC32MatchMySQLSingleTableDataCalculator    | 给 CRC32_MATCH 数据一致性校验算法使用的单表数据计算算法。适用于 MySQL          |
```
Comment (taojintianxia) [inline docs/document/content/dev-manual/scaling.cn.md:50]: why not there is no ‘。’ period  ?

**Comment by taojintianxia** [comment]
this is a high quality PR for documentation, thanks for your contributions 👍

File: docs/document/content/dev-manual/scaling.cn.md (hunk 2 above)
Comment (taojintianxia) [reply docs/document/content/dev-manual/scaling.cn.md:50]: ok

**Review by zjcnb — APPROVED** [review]

**Pull request merged**